packages/react-widgets/src/NumberPicker.tsx

```diff
@@ -132,6 +132,92 @@ const defaultProps = {
   step: 1,
 }
 
+export interface NumberPicker extends Omit<React.HTMLAttributes<HTMLDivElement>, "onChange"> {
```

Comment:
widget props should probably used something out of shared.ts

Reply:
using `export interface NumberPickerProps extends WidgetHTMLProps, Omit<WidgetProps, "onChange">` now